Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -6981,7 +6981,7 @@ def initialize_instance(args: Dict[str, str], params: Dict[str, str]):
         raise DemistoException('Set a port for the instance')
 
     URL = params.get('server', '').rstrip('/:') + ':' + params.get('port', '') + '/api/'
-    API_KEY = str(params.get('key'))
+    API_KEY = str(params.get('key')) or (params.get('credentials') or {}).get('password')  # type: ignore
```

Comment:
```suggestion
    API_KEY = str(params.get('key')) or str((params.get('credentials') or {}).get('password', ''))
```

Reply:
Need the `type: ignore` here to ignore both `call-overload` and `union-attr`

Reply:
Need the ignore here to ignore both `call-overload` and `union-attr`